Library/Homebrew/cmd/migrate-linux-core.rb

```diff
@@ -0,0 +1,48 @@
+# typed: false
+# frozen_string_literal: true
+
+require "utils/popen"
+
+module Homebrew
+  extend T::Sig
+
+  MIGRATION_CACHE_FILE = (HOMEBREW_CACHE/"linuxbrew-core-migration-done").freeze
+
+  module_function
+
+  def migrate_linux_core
+    # Don't run the migration again if it is already done
+    return if File.exist?(MIGRATION_CACHE_FILE)
+
+    formulae_to_migrate = []
+
+    # TODO: Fill final list below just before the brew release
+    linuxbrew_core_versions =
```

Comment:
What is this list? What will be in it and why?

Reply:
I will manually populate it once, based on `git grep "OS\.mac\?"` on the linuxbrew-core repo. This gives a list of 118 formulae right now.

Reply:
That's what will be in the list but I don't understand "why" this list exists.